### PR TITLE
Marya Chuong - HTML Forms exercise

### PR DESCRIPTION
**Pull request by maryalenn**
Screenshot html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <form action="#" method="#" id="coding-form">
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="text" name="email" id="email">
+
+      <div>
+        <label for="why">Why is coding cool?</label>
+      </div>
+      <select name="why">
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure to add an `id` attribute which matches the `for` attribute on the select input

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+
+      <div>
+        <label for="language"><p>Favorite Language (choose one)</p></label>
+      </div>
+      <div>
+        <input type="radio" name="language" value="HTML"> HTML
```
Comment (BrettAlbright) [inline html-forms/index.html]: you will need a label for each input, its common to wrap them around radio inputs like this: 
```html
      <label>
        <input type="radio" name="favoriteLanguage" value="JavaScript">
        JavaScript
      </label>
```

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        <input type="radio" name="language" value="CSS">CSS
+        <input type="radio" name="language" value="JavaScript">JavaScript
+      </div>
+
+      <div>
+        <label for sign-up p"><p>Would you like to sign up for more information?</p></label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: need to fix this `for` attribute

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
a couple small updates, looking good!

File: html-forms/index.html (hunk 2 above)
Comment (maryalenn) [reply html-forms/index.html]: For the radio buttons, do the labels need a for attribute as well, or just < label >?
Screenshot 2022-01-27 201548

File: html-forms/index.html (hunk 1 above)
Comment (maryalenn) [reply html-forms/index.html]: image

File: html-forms/index.html (hunk 2 above)
Comment (maryalenn) [reply html-forms/index.html]: image

File: html-forms/index.html (hunk 3 above)
Comment (maryalenn) [reply html-forms/index.html]: image

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**